packages/backend/src/models/ValidationModel/ValidationModel.ts

```diff
@@ -158,6 +163,10 @@ export class ValidationModel {
                 `${SpaceTableName}.space_uuid`,
             ])
             .orderBy([
+                {
+                    column: `${ValidationTableName}.error`,
+                    order: 'asc',
+                },
```

Comment:
This is going to ruin the order by dashboard name. And having a distinct just for the error column would remove the same error across multiple dashboards.

Changes:
- distinct on name + dashboard_id + error
- order by name, dashboard_id, error

Reply:
got it, thanks!

Reply:
before your requested changes ('another' dashboard didn't show up bc it shared the same error as the other one)

![Screenshot 2023-05-24 at 14 59 47](https://github.com/lightdash/lightdash/assets/7611706/b485db02-5b7b-48d1-b35e-dc3b38fbd913)

now the errors for "another" appears:
![Screenshot 2023-05-24 at 15 16 58](https://github.com/lightdash/lightdash/assets/7611706/8ffbfcd2-667e-4768-8ccd-0531d9826790)